src/styles/card.scss

```diff
@@ -0,0 +1,109 @@
+html{
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center; /* Центрування по горизонталі */
+  background-color: darkgray; /* Колір фону */
+  height: 100vh;
+  align-items: center;
+}
+
+.product-card {
+  background-color: $main-page-color;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+
+  &__image {
+    background-repeat: no-repeat;
+    background-position: center top;
+    margin-top: 32px;
+    background-image: url("../images/imac.jpg");
+    height: 134px;
+    width: 100%;
+  }
+
+
+  &__content {
+    margin: 0 16px;
+  }
+}
+
+  .main__text {
+    width: 166px;
+    font-size: 12px;
+    margin: 0;
+    padding-block: 40px 0;
+    line-height: 18px;
+    color: $text-color;
+    font-weight: 500;
+  }
+
+  .text__code {
+    margin: 0;
+    padding-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-text-color;
+  }
+
+  .content__mark {
+    padding-top: 16px;
+    height: 16px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+    .mark__reviews {
+      color: $text-color;
+      font-size: 10px;
+      line-height: 14px;
+    }
+
+
+  .content__price {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 24px;
+  }
+  .content__price {
+    color: $secondary-text-color;
+    font-size: 12px;
+    line-height: 18px;
+    }
+
+
+  .price__value {
```

Comment:
The class name `.price__value` should follow the BEM naming convention. Consider using `.product-card__content__price__value`.